cost-analyzer/values-instance-types.yaml

```diff
@@ -1,5 +1,7 @@
 instanceTypes:
-  enabled: true
+  # kubectl create secret generic instances-gcp-secret -n kubecost --from-file=secret.json=secret.json
+  gcp:
+    instancesCredential: instances-gcp-secret
   #  kubectl create configmap -n kubecost custom-instance-types --from-file custom.json
   custom:
     configMapName: custom-instance-types
```

Comment:
I followed this template based on [CSV pricing](https://github.com/kubecost/cost-analyzer-helm-chart/pull/3898/files) but URI seems like something we can avoid.  The values-instance-types.yaml can be just

```
instanceTypes:
  # kubectl create secret generic instances-gcp-secret -n kubecost --from-file=secret.json=secret.json
  gcp:
    instancesCredential: instances-gcp-secret
  #  kubectl create configmap -n kubecost custom-instance-types --from-file custom.json
  custom:
    configMapName: custom-instance-types
```